### PR TITLE
Update 163.com.xml

### PR DESCRIPTION
**Pull request by gloomy-ghost**
Add `fankui.`
  Remove commented targets which already exist in top comment

### CODE REVIEW

**Comment by ivysrono** [comment]
奇怪的事情在于，这两条规则在我这不起作用，依然会强制重定向到https

**Comment by gloomy-ghost** [comment]
The exclusions is working on my Firefox, have you disabled the build-in ruleset before testing?

**Comment by ivysrono** [comment]
My mistake……

https://imgsize.ph.126.net/?imgurl=http://lovepicture.nosdn.127.net/-1909745056204601718?imageView_90x90x0x85.net/-1909745056204601718?imageView

**Comment by ivysrono** [comment]
奇怪，本地规则一直都是能覆盖内置规则的，只要名字一样。为什么这次是两者同时生效呢？……

File: src/chrome/content/rules/163.com.xml [hunk 1]
```diff
@@ -117,11 +117,17 @@
...
+		<!-- MCB -->
 		<exclusion pattern="^http://dl\.reg\.163\.com/webzj/m163_1\.0\.1/pub/index_dl\.html" />
 		<test url="http://dl.reg.163.com/webzj/m163_1.0.1/pub/index_dl.html?wdaId=" />
+		<!-- CORS -->
+		<exclusion pattern="^http://dl\.reg\.163\.com/ini" />
+		<test url="http://dl.reg.163.com/ini?pd=mail163&#x26;pkid=CvViHzl&#x26;pkht=mail.163.com" />
 	<target host="passport.126.com" />
 		<exclusion pattern="^http://passport\.126\.com/webzj/m126_1\.0\.1/pub/index_dl\.html" />
 		<test url="http://passport.126.com/webzj/m126_1.0.1/pub/index_dl.html?wdaId=" />
+		<exclusion pattern="^http://passport\.126\.com/dl/ini" />
```
Comment (ivysrono) [inline src/chrome/content/rules/163.com.xml]:
```
<exclusion pattern="^http://passport\.126\.com/dl/(gt|ini)" />
<test url="http://passport.126.com/dl/gt?un=username" />
```

Comment (gloomy-ghost) [inline src/chrome/content/rules/163.com.xml]: ?

Comment (gloomy-ghost) [inline src/chrome/content/rules/163.com.xml]: 刚刚打开163扑面而来的电信劫持……

先折腾投诉的事情去了（逃

Comment (ivysrono) [inline src/chrome/content/rules/163.com.xml]: 我给你提了个pr，cookie那里你可以看下有没有禁用以外的办法。
电信我投诉过，基本没用。
联通威胁他们要投诉就解决问题了。

Comment (ivysrono) [inline src/chrome/content/rules/163.com.xml]: Adblock Plus 中文社区
http://abpchina.org/forum/forum.php

ABP的社区比这里好多了……

Comment (gloomy-ghost) [inline src/chrome/content/rules/163.com.xml]: 我订阅了一堆规则，懒得去找具体哪个的问题了……上京东的时候开incognito就行

**Comment by gloomy-ghost** [comment]
@jeremyn @J0WI @fuglede
Hello guys, I'm sorry for disturbing but could you have a look at this PR? There are some login issues of the largest email provider in China. Both of `163.com` and `126.com` are alexa top-1k, so it would be great if this PR could have some extra attention.

**Comment by jeremyn** [comment]
@gloomy-ghost Can you narrow down the critical parts into a separate, shorter pull request?

**Comment by ivysrono** [comment]
@jeremyn #7944

**Pull request closed without merging**